src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,178 @@
+      }]
+    }}
+  >
+    {DisplayTemplate.bind({})}
+  </Story>
+</Canvas>
+
+### Props
+
+<ArgsTable of={LoginOptionsDisplay} />
+
+## Functional
+
+The `LoginOptions` component handles the 'smart' part of the component. It separates the login options between
+the ones that can be used to 'login as yourself' and the ones that can be used to log in 'on behalf of someone else'.
+It also formats the labels that will be displayed on the buttons and creates the right URL to which the user will be
+redirected upon clicking one of the buttons.
+
+export const Template = ({form, onFormStart}) => (
+  <LiteralsProvider literals={{'beginText': {resolved: 'Begin Form'}}}>
+    <LoginOptions
+      form={form}
+      onFormStart={onFormStart}
+    />
+  </LiteralsProvider>
+);
+
+<Canvas>
+  <Story
+    name="Functional"
+    args={{
+      form: {
+        loginOptions: [
+          {
+            identifier: "digid",
+            label: "DigiD",
+            url: "http://localhost:8999/auth/form-slug/digid/start",
+            logo: {
+              title: "DigiD simulatie",
+              imageSrc: "/digid.png",
+              href: "https://www.digid.nl/"
+            },
+            isForGemachtigde: false
+          },
+          {
+            identifier: "eherkenning",
+            label: "eHerkenning",
+            url: "http://localhost:8999/auth/form-slug/eherkenning/start",
+            logo: {
+              title: "eHerkenning",
+              imageSrc: "/eherkenning.png",
+              href: "https://www.eherkenning.nl/"
+            },
+            isForGemachtigde: false
+          },
+          {
+            identifier: "eidas",
+            label: "eIDAS",
+            url: "http://localhost:8999/auth/form-slug/eidas/start",
+            logo: {
+              title: "eIDAS",
+              imageSrc: "/eidas.png",
+              href: "https://digital-strategy.ec.europa.eu/en/policies/eu-trust-mark"
+            },
+            isForGemachtigde: false
+          },
+          {
+            identifier: "digid_machtigen",
+            label: "DigiD Machtigen",
+            url: "http://localhost:8999/auth/form-slug/digid_machtigen_oidc/start",
+            logo: {
+              title: "DigiD Machtigen",
+              imageSrc: "/digid.png",
+              href: "https://www.digid.nl/"
+            },
+            isForGemachtigde: true
+          },
+          {
+            identifier: "eherkenning_bewindvoering",
+            label: "eHerkenning bewindvoering",
+            url: "http://localhost:8999/auth/form-slug/eherkenning_bewindvoering_oidc/start",
+            logo: {
+              title: "eHerkenning bewindvoering",
+              imageSrc: "/eherkenning.png",
+              href: "https://www.eherkenning.nl/"
+            },
+            isForGemachtigde: true
+          },
+        ]
+      },
+      onFormStart: () => {},
```

Comment:
this now submits for real, leading to a nonsense error/page, so I'd also implement an `event.preventDefault()` and `alert` strategy here purely for storybook purposes

I'd also expect this to be picked up as an interaction which could be tested via the `play` function :-) For an example of such tests, see: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/LanguageSelection/LanguageSelection.stories.mdx?plain=1#L117